lib/management.js

```diff
@@ -0,0 +1,424 @@
+const async = require('async');
+const forge = require('node-forge');
+const arsenal = require('arsenal');
+const config = require('../conf/Config');
+const AWS = require('aws-sdk');
+const werelogs = require('werelogs');
+const bucketclient = require('bucketclient');
+
+werelogs.configure({
+    level: config.log.logLevel,
+    dump: config.log.dumpLevel,
+});
+const logger = new werelogs.Logger('mdManagement');
+
+const serviceBucket = 'PENSIEVE';
+
+const Metadata = arsenal.storage.metadata.MetadataWrapper;
+let metadata;
+
+const params = {
+    bucketdBootstrap: ['localhost'],
+    bucketdLog: null,
+    https: null,
+    metadataClient: {
+        host: config.queuePopulator.dmd.host,
```

Comment:
This may need rework since dmd is not used anymore (e.g. if `config.queuePopulator.dmd` section does not exist it does not start).